hack/tests/e2e-ansible.sh

```diff
@@ -18,22 +18,14 @@ setup_envs $tmp_sdk_root
 
 deploy_operator() {
     header_text "Running deploy operator"
-    kubectl create -f "$OPERATORDIR/deploy/service_account.yaml"
-    kubectl create -f "$OPERATORDIR/deploy/role.yaml"
-    kubectl create -f "$OPERATORDIR/deploy/role_binding.yaml"
-    kubectl create -f "$OPERATORDIR/deploy/crds/ansible.example.com_memcacheds_crd.yaml"
-    kubectl create -f "$OPERATORDIR/deploy/crds/ansible.example.com_foos_crd.yaml"
-    kubectl create -f "$OPERATORDIR/deploy/operator.yaml"
+    kustomize build --load_restrictor none config/testing | kubectl apply -f -
+    kubectl create clusterrolebinding memcached-operator-system-metrics-reader --clusterrole=osdk-metrics-reader --serviceaccount=default:default
 }
 
 remove_operator() {
     header_text "Running remove operator"
-    kubectl delete --wait=true --ignore-not-found=true --timeout=2m -f "$OPERATORDIR/deploy/crds/ansible.example.com_memcacheds_crd.yaml"
-    kubectl delete --wait=true --ignore-not-found=true --timeout=2m -f "$OPERATORDIR/deploy/crds/ansible.example.com_foos_crd.yaml"
-    kubectl delete --wait=true --ignore-not-found=true -f "$OPERATORDIR/deploy/operator.yaml"
-    kubectl delete --wait=true --ignore-not-found=true -f "$OPERATORDIR/deploy/service_account.yaml"
-    kubectl delete --wait=true --ignore-not-found=true -f "$OPERATORDIR/deploy/role.yaml"
-    kubectl delete --wait=true --ignore-not-found=true -f "$OPERATORDIR/deploy/role_binding.yaml"
+    kubectl delete --ignore-not-found clusterrolebinding memcached-operator-system-metrics-reader
+    kustomize build --load_restrictor none config/testing | kubectl delete --wait=true --ignore-not-found=true -f -
```

Comment:
same as above. Also, see that we need to apply the CR is a diff namespace to ensure the Cluster Scope permissions. 

https://github.com/operator-framework/operator-sdk/blob/c1b615c6f01d303ece5bcb06f46565ae0b6b7999/hack/tests/e2e-helm.sh#L29-L34

Reply:
If the ansible e2e wasn't doing that before, it isn't a hard requirement to update it to do it now.  We're simply trying to port the existing e2e to use the new plugin implementation.